sources/packages/backend/libs/utilities/src/base64-utils.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Encode a given string to Base64 string.
+ * @param stringToEncode string content to encode.
+ * @returns Base64 encoded string.
+ */
+export function base64Encode(stringToEncode: string): string {
+  return Buffer.from(stringToEncode, "binary").toString("base64");
```

Comment:
The "binary" does not seem accurate since we are building from a string, Am I missing something?
Can we just use the default utf8 as `Buffer.from(stringToEncode).toString("base64");`?

Reply:
You are right. It works with `Buffer.from(stringToEncode).toString("base64");` will change